src/dhtproto/client/legacy/internal/request/model/IBulkGetRequest.d

```diff
@@ -139,6 +143,20 @@ abstract private class IBulkGetRequest : IChannelRequest, IStreamInfo
                 this.resources.request_suspender);
         }
 
+        // deprecated: remove in dhtproto v15
+        static if (hasFeaturesFrom!("swarm", 5, 2))
```

Comment:
This should actually check for v5.3.x (https://github.com/sociomantic-tsunami/swarm/pull/365 will be released in v5.3.0.)